### ISSUE TITLE
Webcam not working

### ISSUE THREAD

**Issue by Kruemelmann** [tobhe/ubuntu-asahi#130]
Hello, 

I installed Ubuntu again today on my Macbook Pro M1 Max. My camera was and is still not working on Ubuntu 24. Do you have any ideas on this or is this known because the support of asahi is not ported yet?

many thanks for the wonderful project, by the way

**Comment by tobhe** [comment]
Hi @Kruemelmann! The camera is expected to work, I just noticed that there seems to be a bug with our pre-installed camera app though.
Have you tried using it from your browser? Can you see a `/dev/video0` device on your machine?

**Comment by Kruemelmann** [comment]
Yes, you are right, inside the browser the camera is working fine. In the shipped camera app it does not show up.

**Comment by tobhe** [comment]
Thanks for verifying that! I'll see if I can figure out what's wrong with the built-in camera app.
